ping/main.go

```diff
@@ -206,7 +206,7 @@ func runPing(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
 				// into InfluxDB when the test concludes.
 				//
 				// ping-result is the metric name, and round and peer are tags.
-				point := fmt.Sprintf("ping-result,round=%s,peer=%s", tag, id)
+				point := fmt.Sprintf("ping-result.ms,round=%s", tag)
```

Comment:
We should not have any random fields added as tags, as this will explode the cardinality in InfluxDB.